src/rgw/setup/rgw.py

```diff
@@ -620,6 +629,9 @@ def _logrotate_generic(conf: MappedConf):
             Log.info(f'{const.LOGROTATE_TMPL} file copied to {const.LOGROTATE_CONF}')
         except Exception as e:
             Log.error(f"Failed to configure logrotate for {const.COMPONENT_NAME}. ERROR:{e}")
+        # start cron.d service
+        os.system(f"chmod +x {const.CRON_LOGROTATE}")
+        os.system("/usr/sbin/crond start")
```

Comment:
Can we make SimpleProcess calls instead of os.system?

Reply:
ideally we should use SimpleProcess instead of os.system calls. 
Since this is blocker issue and CFT is waiting for this patch to be merged, please check if we can do it quickly or else we can create separate PR and handle this change.

Reply:
@sachitanands Okay in that case we should atleast make os.system calls inside try catch block
```
try:
    os.system()
except OSError as e:
    raise Error(e.errno, 'Error message %s', kwargs)
```

Reply:
Added try except block for system calls